app/api/db/migrations/versions/2022_09_22_11_13_20_964e99759832_create_user_and_role_tables.py

```diff
@@ -55,25 +55,23 @@ def upgrade():
             server_default=sa.text("now()"),
             nullable=False,
         ),
-        sa.Column("example_pet_id", postgresql.UUID(as_uuid=True), nullable=False),
-        sa.Column("name", sa.Text(), nullable=False),
-        sa.Column("species", sa.Text(), nullable=False),
-        sa.Column("pet_owner_id", postgresql.UUID(as_uuid=True), nullable=True),
+        sa.Column("user_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("role_description", sa.Text(), nullable=False),
         sa.ForeignKeyConstraint(
-            ["pet_owner_id"],
-            ["example_person.example_person_id"],
+            ["user_id"],
+            ["user.user_id"],
         ),
-        sa.PrimaryKeyConstraint("example_pet_id"),
+        sa.PrimaryKeyConstraint("user_id", "role_description"),
     )
     op.create_index(
-        op.f("ix_example_pet_pet_owner_id"), "example_pet", ["pet_owner_id"], unique=False
+        op.f("ix_user_role_role_description"), "user_role", ["role_description"], unique=False
```

Comment:
for constraint naming conventions, can we use suffix rather than prefix and `idx` instead of `ix` (e.g. [as described in this stackoverflow answer](https://stackoverflow.com/questions/4107915/postgresql-default-constraint-names/4108266#4108266)
```suggestion
        op.f("user_role_role_description_idx"), "user_role", ["role_description"], unique=False
```

Reply:
Let me see if there's a way to configure that - this is just the auto-generated format.

Reply:
Oh if it's autogenerated then nvm that's fine

Reply:
Looks like it was pretty simple to adjust it, just need to adjust a metadata value in the base table class.